lib/Security/User/UserChecker.php

```diff
@@ -39,6 +39,10 @@ public function checkPostAuth(UserInterface $user): void
     {
         $this->checkValidUser($user);
 
+        /** @var User $user */
+        $pimcoreUser = $user->getUser();
+        $pimcoreUser->setLastLoginDate(); //set user current login date
```

Comment:
Thank you for the PR!
I'd add something like

https://github.com/pimcore/pimcore/pull/18747/files#diff-d9a11a8fd30a1dadfdfc75fe070ecbfa8e8d590787ff38634143c67cc154d4be

to prevent too many sql update queries when checking user validity or by moving the logic of preventing updates within 15 seconds of difference to https://github.com/pimcore/pimcore/blob/3ffaeac8d9f07f877ea4f469015be1774de447b9/models/User/AbstractUser/Dao.php#L163

Reply:
Good idea, i will adapt it, but I think 60s should be better.